crates/core/component/governance/src/validator_vote/action.rs

```diff
@@ -4,7 +4,7 @@ use penumbra_stake::{GovernanceKey, IdentityKey};
 use penumbra_txhash::{EffectHash, EffectingData};
 use serde::{Deserialize, Serialize};
 
-use crate::vote::Vote;
+use crate::{vote::Vote, MAX_VALIDATOR_VOTE_REASON_LENGTH};
```

Comment:
Maybe we should use this constant in proposal_state.rs too? If it requires changing visibility then that's not worth it, just wondering.